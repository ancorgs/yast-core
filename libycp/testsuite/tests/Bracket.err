Parsed:
----------------------------------------------------------------------
{
  list `l = nil;
  return l[0]:1;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  map `m = $[1:11, "a":"aa", `b:`bb];
  return m["a"]:nil;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  map `m = $[1:11, "a":"aa", `b:`bb];
  return m[2]:2;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  return l[1]:nil;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  return l[3]:42;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  map `m = $[1:l, "a":l, `b:l];
  return m["a", 1]:nil;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  map `m = $[1:l, "a":l, `b:l];
  return m["a"]:nil;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  map `m = $[1:l, "a":l, `b:l];
  return m["b"]:l;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  map `m = $["bus":"parallel", "dev_name":"/dev/lp0"];
  `m = (add (m, "uri", (m["bus"]:"xxx" + m["dev_name"]:"yyy")));
  return (m == $["bus":"parallel", "dev_name":"/dev/lp0", "uri":"parallel/dev/lp0"]);
}
----------------------------------------------------------------------
