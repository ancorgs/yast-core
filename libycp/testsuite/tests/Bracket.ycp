{
    list l = nil;
    return l[0]:1;
}
{
    map m = $[1:11, "a":"aa", `b:`bb];
    return m["a"]:nilstring;
}
{
    map m = $[1:11, "a":"aa", `b:`bb];
    return m[2]:2;
}
{
    list l = [1, "a", `b];
    return l[1]:nilinteger;
}   
{
    list l = [1, "a", `b];
    return l[3]:42;
}   
{
    list l = [1, "a", `b];
    map m = $[1:l, "a":l, `b:l];
    return m["a",1]:nilstring;
}
{
    list l = [1, "a", `b];
    map m = $[1:l, "a":l, `b:l];
    return m["a"]:nillist;
}
{
    list l = [1, "a", `b];
    map m = $[1:l, "a":l, `b:l];
    return m["b"]:l;
}



{
    map m = $["bus":"parallel", "dev_name":"/dev/lp0"];
    m = add (m, "uri", m["bus"]:"xxx" + m["dev_name"]:"yyy");
    return (m == $["bus":"parallel", "dev_name":"/dev/lp0", "uri":"parallel/dev/lp0"]);
}
