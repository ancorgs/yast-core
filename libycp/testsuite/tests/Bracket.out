(1)
("aa")
(2)
("a")
(42)
("a")
([1, "a", `b])
([1, "a", `b])
(true)
